dist/images/ovnkube.sh

```diff
@@ -487,13 +520,13 @@ cleanup-ovs-server () {
 # create the ovnkube_db endpoint for other pods to query the OVN DB IP
 create_ovnkube_db_ep () {
   # delete any endpoint by name ovnkube-db
-  kubectl --server=${K8S_APISERVER} --token=${k8s_token} --certificate-authority=${K8S_CACERT} \
+  kubectl --server=${k8s_apiserver} --token=${k8s_token} --certificate-authority=${k8s_cacert} \
     delete ep -n ${ovn_kubernetes_namespace} ovnkube-db 2>/dev/null
 
   # create a new endpoint for the headless onvkube-db service without selectors
   # using the current host has the endpoint IP
-  ovn_db_host=$(getent ahosts $(hostname) | head -1 | awk '{ print $1 }')
-  kubectl --server=${K8S_APISERVER} --token=${k8s_token} --certificate-authority=${K8S_CACERT} create -f - << EOF
+  ovn_db_host=$(host ${K8S_NODE} | awk '{ print $4}')
```

Comment:
(Do we know that `host` is available with the set of packages installed in the ubuntu build? On Fedora it's in `bind-utils`, which we do appear to be installing from the Dockerfiles.)